### ISSUE TITLE
second knob

### ISSUE THREAD

**Issue by leineveber** [fseehawer/react-circular-slider#49]
Is it possible to add somehow second knob to the track?
(also, I have requirement, that this knob is avalailable to move forwards and backwards, in any direction)
<img width="224" alt="Screenshot 2022-03-17 at 19 51 26" src="https://user-images.githubusercontent.com/75081833/158853153-258ee6cb-9fde-420c-a0f3-c54ea2b996b5.png">

**Comment by fseehawer** [comment]
No, currently it's not possible. I am considering implementing it in the near future.

**Comment by mettz** [comment]
Hi @fseehawer, any update on this?

**Comment by jzakotnik** [comment]
This would be awesome, because one could use it as a much better time of day range than datepickers and alike
